server/api/orphaned-record/orphaned-record-action.model.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Entity,
+  Column,
+  CreateDateColumn,
+  BaseEntity,
+  JoinColumn,
+  ManyToOne,
+  BeforeInsert,
+  PrimaryColumn,
+} from 'typeorm';
+import { timestampColumnTransformer } from '../../util/util';
+import { Org } from '../org/org.model';
+import { User } from '../user/user.model';
+
+export enum ActionType {
+  Claim = 'claim',
+  Ignore = 'ignore',
+}
+
+@Entity()
+export class OrphanedRecordAction extends BaseEntity {
+  @PrimaryColumn()
+  id!: string;
+
+  @ManyToOne(() => User, user => user.edipi, {
+    cascade: true,
+    primary: true,
+  })
+  @JoinColumn({
+    name: 'user_edipi',
+  })
+  user?: User;
+
+  @CreateDateColumn({
+    type: 'timestamp',
+    transformer: timestampColumnTransformer,
+  })
+  createdOn!: Date;
+
+  @Column({
+    type: 'timestamp',
+    transformer: timestampColumnTransformer,
+    default: () => `now() + interval '1 day'`,
```

Comment:
It looks like `expiresOn` is being set manually when this model is created. Could we get away with removing the default? I'm just worried we might forget to set it and end up with an unexpected expiration time.